### DISCUSSION TITLE
Avalonia app with Xamarin.Mac SDK

### DISCUSSION THREAD

**Discussion by qt-kaneko** [AvaloniaUI/Avalonia#9440] [Q&A] [answered]
If I make `<TargetFramework>net6.0-macos</TargetFramework>`, I receive this `/usr/local/share/dotnet/sdk/7.0.100/Microsoft.Common.CurrentVersion.targets(894,7): error MSB4057: The target "_CollectBundleResources" does not exist in the project. [/Users/####/####.csproj]`.

So, how can I use Xamarin.Mac SDK in Avalonia app?

P.S. # is just "censorship", imagine that there is some path.

**Comment by maxkatz6** [comment, marked as answer]
Double check if you have RuntimeIdentifier set and you possibly also need a SelfContained property set.
Info.plist file should be optional, but I would add it as well. OutputType shouldn't be WinExe also IIRC.

Here I just created a sample project: https://github.com/maxkatz6/avalonia-net6-macos-sample
Note, I haven't tested this specific project on macOS, but I did the same multiple times before with production apps. If I haven't forgot anything, should work just fine.

**Reply by qt-kaneko** [reply]
Wow, works perfect, thank you a lot.
